src/main/java/org/folio/edge/sip2/repositories/FeeFinesRepository.java

```diff
@@ -0,0 +1,80 @@
+package org.folio.edge.sip2.repositories;
+
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import javax.inject.Inject;
+import org.folio.edge.sip2.session.SessionData;
+
+/**
+ * Provides interaction with the feefines service.
+ *
+ * @author mreno-EBSCO
+ *
+ */
+public class FeeFinesRepository {
+  private final IResourceProvider<IRequestData> resourceProvider;
+
+  @Inject
+  FeeFinesRepository(IResourceProvider<IRequestData> resourceProvider) {
+    this.resourceProvider = Objects.requireNonNull(resourceProvider,
+        "Resource provider cannot be null");
+  }
+
+  /**
+   * Get a patron's manual blocks.
+   *
+   * @param userId the user's ID
+   * @param sessionData session data
+   * @return the manual blocks list in raw JSON
```

Comment:
May want to add the meanings of returning a 0 sized array vs null here. 0 can mean successfully getting manual blocks for the user, whereas null means can't find blocks of the user.

Reply:
I'll update the Javadoc, though, at some point, we will need to revisit how we want to handle errors. What does it mean if we cannot retrieve the manual blocks? Right now, it means that the patron effectively doesn't have any blocks, when, in fact, they might, we just couldn't retrieve them.

I added rudimentary error handling to checkout in `CirculationRepository` to show some error messages in the screen messages field. My plan was to revisit this after title level requests for the other commands. Though, these FOLIO error messages might be too specific for end users.